CHANGELOG.develop

```diff
@@ -639,6 +660,8 @@ is enabled
   Rich Alesi)
 - Set =ivy-use-selectable-prompt= to =t= (thanks to Boris Buliga)
 - Rename git dir variable to upstream changes (thanks to bmag)
+- Add =ivy-enable-advanved-buffer-information= variable to toggle =ivy-rich=
+being enabled (thanks to Alejandro Catalina)
```

Comment:
Typo: "advanved".  Also, the indentation is inconsistent with the rest of the file—when wrapping a line with a list item, the continuation should be indented.

Reply:
yes! the "b" in being should inline with "A" Add in line 663 :)